public/app/features/dashboard-scene/serialization/sceneVariablesSetToVariables.ts

```diff
@@ -104,6 +104,22 @@ export function sceneVariablesSetToVariables(set: SceneVariables) {
         },
         query: variable.state.value,
       });
+    } else if (sceneUtils.isGroupByVariable(variable)) {
+      variables.push({
+        ...commonProperties,
+        datasource: variable.state.datasource,
+        // Only persist the statically defined options
+        options: variable.state.defaultOptions?.map((option) => ({
+          text: option.text,
+          value: String(option.value),
+        })),
+        current: {
+          // @ts-expect-error
+          text: variable.state.text,
```

Comment:
what's the deal with this type mismatch? 🤔

Reply:
It's because the value in state scene can be other than just string or string[].